integration/erc20contract/ObsERC20.sol

```diff
@@ -78,12 +86,20 @@ contract ObsERC20 is ERC20, ContractTransparencyConfig {
     }
 
     function visibilityRules() public override pure returns (VisibilityConfig memory){
-        EventLogConfig[]  memory configs = new EventLogConfig[](1);
-        // erc20 transfer
-        configs[0] = EventLogConfig(hex"ddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef", false, true, true, false, false);
-        return VisibilityConfig(false, configs);
+        bytes32 eventSig = hex"ddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef";
```

Comment:
What is this hash from? 

you can do "var visibilityRulesEventSignature = keccak256("preimage")"